docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md

```diff
@@ -0,0 +1,17 @@
+# Configurations
+
+This document gives some introduction to configurations in Nebula Graph.
+
+## Get configurations
+
+By default, Nebula Graph gets configurations from the Meta Service. But we suggest that you get configurations from local. To get configurations from local, follow these steps:
```

Comment:
By default, Nebula Graph gets configurations from the Meta Service. But we suggest that you get configurations from local if you're new to Nebula Graph.

Reply:
then to do if you are not new? I think this is the only way so far.